### PR TITLE
Fix parameters of internal event function

### PR DESCRIPTION
**Pull request by mjerkov**
Resolves Issue [#395](https://github.com/LiskHQ/lips/issues/395).

### CODE REVIEW

File: proposals/lip-0052.md [hunk 1]
```diff
@@ -1843,6 +1847,10 @@ def transferCrossChain(
         emitFailedTransferCrossChainEvent(senderAddress, nftID, amount, receivingChainID, recipientAddress, INVALID_RECEIVING_CHAIN)
         raise Exception("Receiving chain cannot be the sending chain.")
 
+    if not Interoperability.isChannelActive(receivingChainID):
+        emitFailedTransferCrossChainEvent(senderAddress, nftID, amount, receivingChainID, recipientAddress, RECEIVING_CHAIN_NOT_ACTIVE)
+        raise Exception("Receiving chain is not active.")
```
Comment (vjaiman) [inline proposals/lip-0052.md:1852]: `  emitFailedTransferCrossChainEvent` function doesn't take `amount` as input parameter. Similar check should be done for other verify conditions.

**Review by vjaiman — CHANGES_REQUESTED** [review]

File: proposals/lip-0052.md (hunk 1 above)
Comment (mjerkov) [reply proposals/lip-0052.md:1852]: You're correct, I copy/pasted it from LIP0051. Thanks for noticing it!
Please check this [commit](https://github.com/LiskHQ/lips/pull/396/commits/c6365e48b27ec428cd81d1cf54f8719d8a8055aa).

File: proposals/lip-0052.md [hunk 2]
```diff
@@ -1840,11 +1844,15 @@ def transferCrossChain(
 ) -> None:
 
     if receivingChainID == OWN_CHAIN_ID:
-        emitFailedTransferCrossChainEvent(senderAddress, nftID, amount, receivingChainID, recipientAddress, INVALID_RECEIVING_CHAIN)
+        emitFailedTransferCrossChainEvent(senderAddress, recipientAddress, nftID, receivingChainID, INVALID_RECEIVING_CHAIN)
```
Comment (ricott1) [inline proposals/lip-0052.md:1847]: This check is superfluous now

**Review by ricott1 — CHANGES_REQUESTED** [review]
- In the `emitFailedTransferCrossChainEvent` function you have the wrong parameters order: NFTID should be before receivingChainID.

File: proposals/lip-0052.md (hunk 2 above)
Comment (mjerkov) [reply proposals/lip-0052.md:1847]: As discussed, it's not wrong to keep this check for reasons of clarity.

**Review by ricott1 — APPROVED** [review]

**Review by Incede — APPROVED** [review]

File: proposals/lip-0052.md [hunk 3]
```diff
@@ -188,6 +188,7 @@ The following constants are used throughout the document:
 | `RESULT_DATA_TOO_LONG` | uint32 | 13 | Used when the data input is too long. |
 | `INVALID_RECEIVING_CHAIN` | uint32 | 14 | Used when, during the cross-chain token transfer, the receiving chain is set to be equal to the sending chain. |
 | `RESULT_INVALID_ACCOUNT` | uint32 | 15 | Used when recover function fails due to invalid account. |
+| `RECEIVING_CHAIN_NOT_ACTIVE` | uint32 | 16 | Used when, during the cross-chain token transfer, the receiving chain is not active. |
```
Comment (janhack) [inline proposals/lip-0052.md:191]:
```suggestion
| `RECEIVING_CHAIN_NOT_ACTIVE` | uint32 | 16 | Used when, during the cross-chain NFT transfer, the receiving chain is not active. |
```

**Review by janhack — APPROVED** [review]
Editorial approval

**Pull request merged**